.codespell.exclude

```diff
@@ -105,3 +105,4 @@ system Currus
 		"i' amazin"          # is amazing (mispronounced)
             * Care Package 2a has been replaced with a new mission, FW Deep Memorial, a set of missions and events that offers regardless of the player's choices, but which has a branch if Brower dies during the attack.
 			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'im and he can get'cha and this stone where it needs to go." The young man then looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
+		"Love's Labour Lost"	# The standard English spelling of the play's name.
```

Comment:
```suggestion
		"Love's Labour Lost"
```
The checks will continue to fail unless the line is exactly the same as the one you want to exclude.